docs/run/sessions.mdx

```diff
@@ -28,13 +28,15 @@ There are several benefits to using sessions:
 
 The basic workflow for sessions is as follows:
 
-1. The first job in a session enters the normal queue. 
-2. When the first job starts running, the _maximum timeout_ clock starts. 
-2. Subsequent jobs within the session are prioritized over others, reducing wait times.
-3. The _interactive timeout_ runs between the jobs in a session. Every session has an interactive timeout value (ITTL, or interactive time to live). If there are no session jobs queued within the ITTL window, the session is temporarily deactivated and normal job selection resumes. A deactivated session can be resumed for the next job<sup>*</sup> if the session has not reached its maximum timeout value.
-3. If the maximum timeout value is reached, the sessions end and any remaining queued jobs fail.
+1. The first job in a session enters the normal queue. The job's owner become the priority user.
```

Comment:
Why the mention of priority user? I don't see it mentioned again anywhere.

Reply:
> Why the mention of priority user? I don't see it mentioned again anywhere.

Oh, I think it was to match what's in the video.  But I can remove it.